muser/src/main/java/edu/usf/cutr/mfe/manager/MuserMusicDataAnalysisManager.java

```diff
@@ -0,0 +1,117 @@
+package edu.usf.cutr.mfe.manager;
+import com.google.cloud.firestore.QueryDocumentSnapshot;
+import edu.usf.cutr.mfe.exception.FirebaseFileNotInitializedException;
+import edu.usf.cutr.mfe.io.CSVFileWriter;
+import edu.usf.cutr.mfe.io.FirebaseReader;
+import edu.usf.cutr.mfe.model.MusicAnalysisModel;
+import edu.usf.cutr.mfe.options.ProgramOptions;
+import java.util.ArrayList;
+import java.util.List;
+
+public class MuserMusicDataAnalysisManager {
+	private CSVFileWriter mCSVFileWriter;
+	private FirebaseReader mFirebaseReader;
+	private ProgramOptions mProgramOptions;
```

Comment:
Let's remove the `m` prefix from all the class-level variables. For example, it should be `programOptions`.